package.json

```diff
@@ -51,6 +51,7 @@
     "mapbox-gl": "^0.12.0"
   },
   "dependencies": {
+    "event-emitter": "^0.3.4",
```

Comment:
any by that, i mean `require('events')` without requiring a new module, which should work cc @tristen